src/main/java/frc/robot/subsystems/intake/IntakeIOSim.kt

```diff
@@ -0,0 +1,43 @@
+package frc.robot.subsystems.intake
+
+import edu.wpi.first.hal.SimBoolean
+import edu.wpi.first.math.system.plant.DCMotor
+import edu.wpi.first.math.util.Units
+import edu.wpi.first.units.Measure
+import edu.wpi.first.units.MutableMeasure
+import edu.wpi.first.units.Units.Volts
+import edu.wpi.first.units.Voltage
+import edu.wpi.first.wpilibj.RobotController
+import edu.wpi.first.wpilibj.simulation.FlywheelSim
+import lib.math.units.into
+
+class IntakeIOSim : IntakeIO {
+    private val intakeSim: FlywheelSim = FlywheelSim(
+        DCMotor.getNeo550(1),
+        2.0,
+        0.0000719
+    )
```

Comment:
What is all this?

Reply:
Agreed; magic numbers are bad

Reply:
Gearing, and MOI, ill add those to the constructors, oops